crates/core/tedge_mapper/src/c8y/tests.rs

```diff
@@ -1156,7 +1156,7 @@ async fn mapper_publishes_supported_operations() {
     // and verifies the supported operations that are published by the tedge_mapper.
     let broker = mqtt_tests::test_mqtt_broker();
     let cfg_dir = TempTedgeDir::new();
-    create_thin_edge_operations(&cfg_dir, vec!["c8y_TestOp2", "c8y_TestOp2"]);
+    create_thin_edge_operations(&cfg_dir, vec!["c8y_TestOp1", "c8y_TestOp2"]);
```

Comment:
Is this fixing an actual issue with that test? It seems unrelated to sporadic failures.

Reply:
This is not related to a sporadic test failure issue. But found that it was wrong, so I did fix the typo.